### PR TITLE
upgrade: Rename the progress file, mark the end of whole upgrade

### PR DESCRIPTION
**Pull request by jsuchome**
This will need adaptations elsewhere, e.g. in mkcloud scripts...

### CODE REVIEW

File: crowbar_framework/lib/crowbar/upgrade_status.rb [hunk 1]
```diff
@@ -109,6 +112,9 @@ def end_step(success = true, errors = {})
         }
         next_step
         save
+        if finished? && success
+          FileUtils.touch("/var/lib/crowbar/upgrade/6-to-7-done")
```
Comment (MaximilianMeister) [inline crowbar_framework/lib/crowbar/upgrade_status.rb]: the installer files are all in the schema `crowbar-installed-ok` or `crowbar-install-failed`, shall we stick to this schema? that would be `6-to-7-upgraded-ok`

Comment (jsuchome) [inline crowbar_framework/lib/crowbar/upgrade_status.rb]: -ok :-)

**Comment by jsuchome** [comment]
[(3) Restrict API access during Upgrade](https://trello.com/c/MZP6krgU/117-3-restrict-api-access-during-upgrade)

**Comment by MaximilianMeister** [comment]
I'll adapt https://github.com/crowbar/crowbar-core/pull/861 accordingly

**Pull request merged**